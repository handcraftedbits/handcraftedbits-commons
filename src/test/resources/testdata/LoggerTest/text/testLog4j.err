WARN  | main                 | c.h.c.api.log.LoggerTest  | warn from Log4j 
WARN  | main                 | c.h.c.api.log.LoggerTest  | warn from Log4j with MDC | key1=value1, key2=value2
ERROR | main                 | c.h.c.api.log.LoggerTest  | error from Log4j 
ERROR | main                 | c.h.c.api.log.LoggerTest  | error from Log4j with MDC | key1=value1, key2=value2
