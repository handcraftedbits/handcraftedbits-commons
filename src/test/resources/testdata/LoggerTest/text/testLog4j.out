TRACE | main                 | c.h.c.api.log.LoggerTest  | trace from Log4j 
TRACE | main                 | c.h.c.api.log.LoggerTest  | trace from Log4j with MDC | key1=value1, key2=value2
DEBUG | main                 | c.h.c.api.log.LoggerTest  | debug from Log4j 
DEBUG | main                 | c.h.c.api.log.LoggerTest  | debug from Log4j with MDC | key1=value1, key2=value2
INFO  | main                 | c.h.c.api.log.LoggerTest  | info from Log4j 
INFO  | main                 | c.h.c.api.log.LoggerTest  | info from Log4j with MDC | key1=value1, key2=value2
